### PR TITLE
OP-1184 | Add new inventory for all products or for specific product

### PR DESCRIPTION
**Pull request by ArnaudFonzam**
See OP-1184.

### CODE REVIEW

File: src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public MedicalInventory newMedicalInventory(MedicalInventory medicalinventory) t
 	 * @return the updated {@link MedicalInventory} object.
 	 * @throws OHServiceException
 	 */
-	public MedicalInventory updateMedicalInventoryRow(MedicalInventory medicalInventory) throws OHServiceException {
+	public MedicalInventory updateMedicalInventory(MedicalInventory medicalInventory) throws OHServiceException {
 		return ioOperations.updateMedicalInventoryRow(medicalInventory);
 	}
```
Comment (mwithi) [inline src/main/java/org/isf/medicalinventory/manager/MedicalInventoryManager.java:61]: the called method is still Row?

File: src/main/java/org/isf/medicals/manager/MedicalBrowsingManager.java [hunk 2]
```diff
@@ -59,6 +59,17 @@ public class MedicalBrowsingManager {
 	public Medical getMedical(int code) throws OHServiceException {
 		return ioOperations.getMedical(code);
 	}
+	
+	/**
+	 * Returns the requested medical.
+	 *
+	 * @param code the medical prod_code.
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/medicals/manager/MedicalBrowsingManager.java:66]:
```suggestion
	 * @param prod_code the medical prod_code.
```
make it match the parameter in the method signature

File: sql/step_a1XX_add_inventory_sub_menu_in_pharmacy.sql
Comment (mwithi) [inline sql/step_a1XX_add_inventory_sub_menu_in_pharmacy.sql:1]: this seems a duplicate of step_a106, isn't it?

Comment (ArnaudFonzam) [reply sql/step_a1XX_add_inventory_sub_menu_in_pharmacy.sql:1]: yes you rigth so which one should I remove please.

Comment (ArnaudFonzam) [reply sql/step_a1XX_add_inventory_sub_menu_in_pharmacy.sql:1]: Okay I have upade it and use  this name `step_a105_add_inventory_sub_menu_in_pharmacy.sql`. also update `step_04_all_following_steps.sql`  to apply this change

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**